docs/xdr/features/collect/integrations/endpoint/ibm_aix.md

```diff
@@ -0,0 +1,30 @@
+uuid: e8cb3372-41c8-409f-8e0e-ca4e6cecf7e6
+name: IBM AIX 
+type: intake
+
+## Overview
+
+AIX (Advanced Interactive eXecutive), is a series of proprietary Unix operating systems developed and sold by IBM for several of its computer platforms.
+
```

Comment:
Recent change on the documentation, to associate rule to datasource, we should include a new autogenerated document
```suggestion
{!_shared_content/operations_center/detection/generated/suggested_rules_e8cb3372-41c8-409f-8e0e-ca4e6cecf7e6_do_not_edit_manually.md!}

```